EduID/localizations.yaml

```diff
@@ -83,6 +83,10 @@ IOS:
         en: Something went wrong (404)
         nl: Er is iets fout gegaan (404).
   Profile:
+    YouAreLoggedIn:
+        COPY:
+          en: You are logged in
+          nl: U bent ingelogd
```

Comment:
Everywhere else we use 'je', is this correct?

Reply:
honestly, I thought let me be polite 😁, I didn't check elsewhere. I'll push a fix right away.